pkg/staticnetworkconfig/generator.go

```diff
@@ -20,6 +20,11 @@ import (
 const staticNetworkConfigHostsDelimeter = "ZZZZZ"
 const hostStaticNetworkDelimeter = "HHHHH"
 
+type Config struct {
+	MaxConcurrentGenerations int `envconfig:"MAX_CONCURRENT_NMSTATECTL_GENERATIONS" default:"30"`
```

Comment:
I'm still investigating how this should all work together. These are all pretty extreme upper limits and they should all be configurable after deploy.

That said, this is a much more expensive operation than anything we're doing directly in the image service because this spawns a new process every request. If there is no static network config or if we're using the full iso this won't get called and the image service would be able to handle more.